### PR TITLE
e2e: add policy-heat-idlepage e2e test 

### PR DESCRIPTION
**Pull request by yanjing1104**
- Support MEME_BRC and MEME_BRS in memtierd-meme-start
- Add e2e test when the policy is heat and tracker is idlepage, using meme to actively read memory

### CODE REVIEW

**Comment by yanjing1104** [comment]
Passed in both ubuntu-22.04 and debian-sid.

File: test/e2e/memtierd.test-suite/policy-heat/n4c16/test00-swap-idlepage/code.var.sh [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Make sure the system has IDLEPAGE support enabled
+# Note: IDLEPAGE is enabled on ubuntu-22.04 by fedault
```
Comment (askervin) [inline test/e2e/memtierd.test-suite/policy-heat/n4c16/test00-swap-idlepage/code.var.sh]: s/fedault/default

**Review by askervin — COMMENTED** [review]
```
PAGEOUT[G]
1.299438
memtierd> grep PAGEOUT value matching 1\.[3-5] not found
```

The test failed on the first run on distro=debian-sid, but it got very very close to passing the final check. On the second run it succeeded.

Maybe we could simply relax the final step to matching `1\.[2-5]`, what do you think?

In general, this is very valuable addition to the test suite! It's very nice to see the idlepages tracker and the heat policy working nicely in swapping. :)

File: test/e2e/memtierd.test-suite/policy-heat/n4c16/test00-swap-idlepage/code.var.sh (hunk 1 above)
Comment (yanjing1104) [reply test/e2e/memtierd.test-suite/policy-heat/n4c16/test00-swap-idlepage/code.var.sh]: Done

**Pull request merged**